artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTSession.java

```diff
@@ -212,4 +234,42 @@ public CoreMessageObjectPools getCoreMessageObjectPools() {
       return coreMessageObjectPools;
    }
 
+   public boolean is5() {
+      return five;
+   }
+
+   public void set5(boolean five) {
```

Comment:
small point, but I am not sure on five and boolean, it may be better to keep it to int and int > 3, I imagine there will be mqtt 6  etc... and they will build on each other.

Reply:
That's a fair point. However, I'd prefer to leave it as a `boolean` for now and change it later if/when necessary. It will be just as easy to change it later as it would be to change it now. In my opinion the benefit of not changing it now is that using a `boolean` is more clear.